src/assets/wise5/components/draw/drawService.ts

```diff
@@ -16,6 +16,24 @@ import { UtilService } from '../../services/utilService';
 
 @Injectable()
 export class DrawService extends ComponentService {
+  toolFieldToLabel: any = {
+    select: 'Select tool',
+    line: 'Line tool (click and hold to show available line types)',
+    shape: 'Basic shape tool (click and hold to show available shapes)',
+    freeHand: 'Free hand drawing tool',
+    text: 'Text tool (click and hold to show available font sizes)',
+    stamp: 'Stamp tool (click and hold to show available categories)',
+    strokeColor: 'Stroke color (click and hold to show available colors)',
+    fillColor: 'Fill color (click and hold to show available colors)',
+    clone: 'Clone tool',
+    strokeWidth: 'Stroke width (click and hold to show available options)',
+    sendBack: 'Send selected objects to back',
+    sendForward: 'Send selected objects to front',
+    undo: 'Undo',
+    redo: 'Redo',
+    delete: 'Delete selected objects'
+  };
```

Comment:
i18n?

Reply:
I originally had these localized but then realized it won't work if they're translated. The way we show/hide tools is by finding the element with the given title since there aren't any API functions to show/hide specific tools. The title values come from within the DrawingTool project. The translation would have to be done within the DrawingTool project or we would have to use some mapping from English to other language and replace the titles programmatically.